draft-protected-headers.md

```diff
@@ -135,9 +135,99 @@ Due to compatibility and usability concerns, a Mail User Agent SHOULD NOT obscur
 
 Aside from that limitation, this specification does not at this time define or limit the methods a MUA may use to convert Exposed Headers into Obscured Headers.
 
-Legacy Display
---------------
+Legacy Display {#legacy-display}
+==============
+
+MUAs typically display some headers directly to the user.
+An encrypted message may be read by an decryption-capable MUA that is unaware of this standard.
+The user of such a legacy client risks losing access to any obscured headers.
+
+This section presents a workaround to mitigate this risk by restructuring the Cryptographic Payload before encrypting to include a "Legacy Display" part.
+
+Typically Visible Headers {#typically-visible-headers}
+-------------------------
+
+Of all the headers that an e-mail message may contain, only a handful are typically presented directly to the user.
+The typically visible headers are:
+
+ - `Subject`
+ - `From`
+ - `To`
+ - `Cc`
+ - `Date`
+
+The above is a complete list.  No other headers are considered "typically visible".
+
+Other headers may affect the visible rendering of the message (e.g., `References` and `In-Reply-To` may affect the placement of a message in a threaded discussion), but they are not directly displayed to the user and so are not considered "typically visible" for the purposes of this document.
+
+Message Generation: Including a Legacy Display Part
+---------------------------------------------------
+
+A generating MUA that wants to make an obscured Subject (or any other typically visible header) visible to a recipient using a legacy MUA SHOULD modify the Cryptographic Payload by wrapping the intended body of the message in a `multipart/mixed` MIME part that prefixes the intended body with a Legacy Display part.
```

Comment:
s/obscured Subject/Obscured Subject/ for consitency with the terminology ?

Reply:
Thanks for the review!

I was capitalizing "Subject" because header names are typically capitalized.  I did not capitalize "Obscured" because i was only capitalizing explicitly-defined terms, and the explicitly-defined term is "Obscured Headers", not "obscured" on its own.

I'm happy to capitalize it if you think it makes more sense.  Maybe the "explicitly-defined" term should just be "Obscured" anyway.